w1r3/go/main.go

```diff
@@ -474,19 +489,19 @@ func enableTracing(ctx context.Context, sampleRate float64, projectID string) (f
 
 func histogramBoundaries() []float64 {
 	boundaries := make([]float64, 0)
-	boundary := 0.0
-	increment := 2.0
+	boundary := 0 * time.Second
+	increment := 2 * time.Millisecond
 	for range 50 {
-		boundaries = append(boundaries, boundary)
+		boundaries = append(boundaries, boundary.Seconds())
 		boundary += increment
 	}
-	boundary = 100.0
-	increment = 10.0
+	boundary = 100 * time.Millisecond
+	increment = 10 * time.Millisecond
```

Comment:
I'm confused with use of milisecond while the PR title is "use seconds"; can you clarify?

Reply:
Yes. The units of the histogram are seconds so 1.0 represents 1 second.  But we still want millisecond-sized buckets, so we want to create boundaries at 0.000, 0.002, 0.004, etc.  Using floating point is scary, and I think confusing.  Saying "use increments of 2 millliseconds" is (in my opinion) clearer.  Added a comment.